<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OllamaChatDockWidgetBase</class>
 <widget class="QDockWidget" name="OllamaChatDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ollama Chat</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_processing">
       <attribute name="title">
        <string>Process</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_apiUrl">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>366</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_fetch">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>366</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_response">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>366</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_execute">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>360</y>
          <width>366</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Execute</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_discussion">
       <attribute name="title">
        <string>Discussion</string>
       </attribute>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>361</width>
          <height>401</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLineEdit" name="lineEdit_discussprompt"/>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_discussask">
           <property name="text">
            <string>Ask</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QTextEdit" name="textEdit_discussresponse"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_settings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>341</width>
          <height>211</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_host">
           <property name="text">
            <string>Host</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_2_hosturl"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2_model">
           <property name="text">
            <string>Model</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBox_modelchoice"/>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_save">
           <property name="text">
            <string>Save settings</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
